app/components/app_consent_component.html.erb

```diff
@@ -7,28 +7,47 @@
 <% end %>
 
 <% if patient_session.consents.present? %>
-  <% consents_grouped_by_parent.each do |summary, consents| %>
-    <%= render AppDetailsComponent.new(summary:, open: open_consents?) do %>
-      <%= render AppConsentSummaryComponent.new(**grouped_consents(consents)) %>
+  <% consents = patient_session.consents.order(:recorded_at) %>
```

Comment:
I think this could be a method inside the component Ruby class:

```ruby
def consents
  patient_session.consents.order(:recorded_at)
end
```

Reply:
Yup, thanks for catching – I changed this component too quickly and missed another clean-up opportunity. Fixed in [b5ba1bd](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/1324/commits/b5ba1bd3a80d7293edf086cc407d10728be8b195)